### ISSUE TITLE
Obsidian's non-standard YAML syntax of list is causing problem in digital-garden

### ISSUE THREAD

**Issue by cyanlink** [oleeskild/obsidian-digital-garden#358]
I'm a fairly new user to obsidian, but I can tell the following syntax of list (only commas as separators, no brackets) is not standard YAML (IIRC only comma separated list in square brackets, or new lines starting with hyphen are correct syntax):
See official doc: https://help.obsidian.md/Editing+and+formatting/Tags
```
tags: list, abcd
```
Such syntax would work with Obsidian's own tag search (i.e. they are considered two tags), but it would cause problem because digital-garden just converting the frontMatter key-value pair into a YAML string.
The rendered outcome would be like:
```
{"dg-publish":true,"permalink":"/game-smuggler/outline/","title":"Outline","tags":["list, abcd"],"created":"2023-07-31T19:02","updated":"2023-07-31T22:00:50.634+08:00"}
```
So basically the tag is recognized as one instead of two.
![image](https://github.com/oleeskild/obsidian-digital-garden/assets/14952800/9605a136-0a8e-4488-abc3-c51fae348b83)

Should we do something special, or keep it as is?

**Comment by jpoles1** [comment]
I'm running in to the same issue. 

Frankly I find that writing tags with the proper yaml array syntax is slow and cumbersome for my workflow. I much prefer just listing my tags as a comma separated list. 
```
tags: tag1, tag2, etc
```

I think it would be beneficial if this formatting would be recognized as well!

**Comment by julesvirallinen** [comment]
Would using the linter plugin to automatically save these as valid YAML make sense for you guys?
